### PR TITLE
:zap: It translation code comments

### PR DESCRIPTION
**Pull request by omartek**
Added translation of code comments.
Added a line at the end with link to etch.io deployment guide.

### CODE REVIEW

**Comment by omartek** [comment]
For link to other translated pages, line at the beginning of polish Jettycat, is better to use:
[Making Games: Jetty Cat](tut-making-jettycat.**html**) or (tut-making-jettycat.**md**)?

thanks

File: docs/it/tut-polishing-jettycat.md [hunk 1]
```diff
@@ -296,4 +302,7 @@ Il menu principale ora avrà un cielo animato e il cielo nel livello di gioco pr
 
 Il gioco è rifinito e sembra fantastico, evviva! È ora di leggere altri tutorial o di creare un nuovo gioco da zero!
 
-Buona codifica!
+Ricorda che puoi pubblicare i tuoi giochi su **itch.io** o piattaforme di giochi on line simili. È facile, [leggi le istruzioni nella guida](deployment-itch-io.html).
```
Comment (CosmoMyzrailGorynych) [inline docs/it/tut-polishing-jettycat.md:305]: Let's not deviate much from the english docs.

**Comment by CosmoMyzrailGorynych** [comment]
> For link to other translated pages, like at the beginning of polish Jettycat, is better to use:
> -Making Games: Jetty Cat-(tut-making-jettycat.html) or (tut-making-jettycat.md)?

I think both work, but usually .html is what is used.

File: docs/it/tut-polishing-jettycat.md (hunk 1 above)
Comment (omartek) [reply docs/it/tut-polishing-jettycat.md]: ok

**Pull request merged**